### PR TITLE
Make only top part of tracker popup draggable

### PR DESCRIPTION
**Pull request by akalin-keybase**
This makes the rest of the popup clickable on Linux.

### CODE REVIEW

File: shared/tracker2/assertion/index.js [hunk 1]
```diff
@@ -396,7 +396,12 @@ class Assertion extends React.PureComponent<Props, State> {
 }
 
 const styles = Styles.styleSheetCreate({
-  container: {flexShrink: 0, paddingBottom: 4, paddingTop: 4},
+  container: Styles.platformStyles({
+    common: {flexShrink: 0, paddingBottom: 4, paddingTop: 4},
+    isElectron: {
+      ...Styles.desktopStyles.windowDraggingClickable,
```
Comment (buoyad) [inline shared/tracker2/assertion/index.js]: :sheep: might be a better separation of concerns if we put this in `styles.assertions` in `tracker2/index.desktop` if that also works.

File: shared/tracker2/bio/index.js [hunk 2]
```diff
@@ -74,7 +74,12 @@ const Bio = (p: Props) => (
 
 const styles = Styles.styleSheetCreate({
   bold: {...Styles.globalStyles.fontBold},
-  container: {backgroundColor: Styles.globalColors.white, flexShrink: 0},
+  container: Styles.platformStyles({
+    common: {backgroundColor: Styles.globalColors.white, flexShrink: 0},
+    isElectron: {
+      ...Styles.desktopStyles.windowDraggingClickable,
```
Comment (buoyad) [inline shared/tracker2/bio/index.js]: same here (though I guess there isn't a place for it currently)

**Comment by chrisnojima** [comment]
we could just make the top draggable and not have to worry about the body

**Comment by akalin-keybase** [comment]
Yeah, making the top draggable makes this much easier. PTAL @buoyad since this is basically a redo!

**Pull request merged**